### ISSUE TITLE
How to use this library with custom _app.js

### ISSUE THREAD

**Issue by ReangeloJ** [vinissimus/next-translate#164]
I tried to follow the steps for dynamic paging with a custom _app.js but then everything breaks and all the props in my components become undefined. Do you know if this library can work with custom app.js?

**Comment by aralroca** [comment]
Yes, it should work with a custom`_app.js`. Can you provide more info or share your project to see more details?
